### ISSUE TITLE
Run multipage a11y assessment tool in HTML

### ISSUE THREAD

**Issue by atsansone** [dart-lang/site-www#5290]
Using pa11y, pull a list of accessibility issues in docs.flutter.dev. These should be human-reviewable in HTML.

Use the following command:

```terminal
pa11y-ci --reporter=pa11y-ci-reporter-html --sitemap https://dart.dev/sitemap.xml
```

**Comment by atsansone** [comment]
[dart-dev.zip](https://github.com/dart-lang/site-www/files/13182118/dart-dev.zip)
